vscode_extension/src/providers/AacDefinitionsViewProvider.ts

```diff
@@ -54,26 +57,59 @@ export class Definition extends vscode.TreeItem {
     private definitionType: string;
 }
 
-function getDefinitionByName(definitionName: string) {
-    return Promise.resolve(aacRestApi.getDefinitionByNameDefinitionGet(definitionName, true));
+export async function createDefinition() {
+    // Prompt the user to pick an AaC file to store the new definition in.
+    const definitionFileOpenDialogOptions: vscode.OpenDialogOptions = {
```

Comment:
I haven't tested it yet, so it may not be applicable (if so, disregard), but will this require the file to already exist before I can add a definition to a new/empty file?

Reply:
You should be able to create a new file to insert into via the prompt to pick a directory.

Reply:
Hmm. Looks like a limitation of the VSCode window API https://code.visualstudio.com/api/references/vscode-api#OpenDialogOptions